ansible/roles/ocp-workload-rhte19-shadowman/tasks/post_workload.yml

```diff
@@ -38,7 +38,10 @@
   debug:
     msg: "user.info: {{ item }}"
   with_items:
-    - "Openshift API for command line 'oc' client: {{ showserver.stdout | trim }}"
+    - "OpenShift API for command line 'oc' client: {{ showserver.stdout | trim }}"
+    - "user.info: Login Info:"
+    - "user.info: Cluster User: opentlc-mgr / r3dh4t1!"
```

Comment:
Please don't hardcode passwords.
This should be a variable.

Reply:
Done. Thanks!